### PR TITLE
TR55 Table Output Columns

### PR DESCRIPTION
**Pull request by mmcfarland**
* Represent the data in the water depth chart also in table form, including
the additional non-modifed column (either 100% Forest or Current
conditions depending on the scenario).
* Include a new column in the runoff table for the volume adjusted to the
AoI size and the Water Quality table for load adjusted by adjusted
runoff volume.

##### To Test
1. Create an AoI and note it's size (`471907` in my screenshots below)
2. Go to the model page and notice a new table for runoff.
  * Check that the values for water depth are consistent with the chart
  * Check that the values for volume are consistent with the formula: `(depth/100) * aoiArea`
3. Go to the water quality tab
  * Check that the new column, Concentration, has values consistent with the formula `load / modified runoff volume (from above)`

screenshot from 2016-01-22 11 36 55

screenshot from 2016-01-22 11 37 11

Connects #1091 
Connects #1092

### CODE REVIEW

**Comment by mmcfarland** [comment]
@caseypt added you since some changes may be relevant to the sidebar change.

**Comment by caseypt** [comment]
I've tried a couple AoI's of different size, and every one has `NaN` for the "Average Concentration" values on the water quality tab.

**Comment by caseypt** [comment]
Sample response here: https://gist.github.com/caseypt/a40a5f831f24100f708b

File: src/mmw/js/src/modeling/tr55/quality/views.js [hunk 1]
```diff
@@ -64,12 +69,28 @@ var ResultView = Marionette.LayoutView.extend({
...
-    template: tableRowTmpl
+    template: tableRowTmpl,
+
+    templateHelpers: function() {
+        var load = this.model.get('load'),
+            runoff = this.model.get('runoff'),
+            adjustedRunoff = this.options.aoiVolumeModel.adjust(runoff);
+
+        return {
+            concentration: load / adjustedRunoff
```
Comment (caseypt) [inline src/mmw/js/src/modeling/tr55/quality/views.js]: `runoff` is always null for me, which is leading to the `NaN` for concentration.

Comment (caseypt) [inline src/mmw/js/src/modeling/tr55/quality/views.js]:
```
this.model.attributes
Object {load: 4573745.559121256, measure: "Biochemical Oxygen Demand"}
```

Comment (caseypt) [inline src/mmw/js/src/modeling/tr55/quality/views.js]: Restarting Celery fixed this. Thanks.

**Comment by caseypt** [comment]
Consider restructuring the runoff table to look more like this:

screen shot 2016-01-25 at 2 49 15 pm

I think this makes it easier to compare the scenarios.

**Comment by caseypt** [comment]
+1. Tested. Nice job.

**Comment by mmcfarland** [comment]
Thanks - I like that layout better.  Stroud defined the table/columns they wanted so I'd like to present them as is, but will suggest this update.

**Pull request merged**